recipe/meta.yaml

```diff
@@ -14,17 +14,17 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: True  # [win]
 
 requirements:
   build:
     - python
     - setuptools
-    - gcc
+    - gcc  # [not win]
```

Comment:
Now that we are building on Windows we need to limit `gcc` (and `libgfortran` below) to Linux and OS X.